Lib/test/test_support.py

```diff
@@ -746,7 +746,18 @@ def test_get_signal_name(self):
 
     def test_linked_to_musl(self):
         linked = support.linked_to_musl()
-        self.assertIsInstance(linked, bool)
+        self.assertIsNotNone(linked)
+        if support.is_wasi:
+            self.assertTrue(linked)
```

Comment:
Check maybe even that it's equal to (0, 0, 0)?

Reply:
I'd rather not...eventually I hope it will be a version number.